.env.template

```diff
@@ -0,0 +1,17 @@
+###
+### Default values of the .env files.
+###
+
+# The domain of the @code-reviewer/api
+DOMAIN=http://localhost
+
+# The port of the @code-reviewer/api
+PORT=3000
+
+# the graphql endpoint for @code-reviewer/api
+GRAPHQL_ENDPOINT=graphql
+
+# MongoDB server endpoint
+MONGO_SERVER=mongodb://localhost:27017/csca08
```

Comment:
I think we should give db a more generic name like code reviewer.

Reply:
DB's name is code_reviewer now